test/cleanup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script looks for orphaned helm releases and namespace in the CI system
+# and cleans them up. It only needs to be run periodically.
+# Orphaned releases are seen as those greater than 5 hours old.
+
+# NOTE: This script is designed to be run in Darwin (MacOS) and GNU/Linux systems.
+# Since date commands can vary between systems there has not been testing outside
+# of these.
+
+# Opt-in to installing Helm if not local
+if [ -n "${INSTALL_HELM}" ]; then
+  HELM_URL=https://storage.googleapis.com/kubernetes-helm
+  HELM_TARBALL=helm-v2.7.2-linux-amd64.tar.gz
```

Comment:
Some variables are all caps, some are all lower case. Can we agree on one style?